### PR TITLE
[feat]: [CDS-42099]: change azure logs style

### PR DESCRIPTION
**Pull request by tmacari-harness**
## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>
  
- Feature build: `trigger feature-build`
- Immutable delegate `trigger publish-delegate`
</details>

<details>
  <summary>PR Check triggers</summary>
  
  You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ti0, ti1`
  
- Compile: `trigger compile`
- CodeFormat: `trigger codeformat`
- MessageMetadata: `trigger messagecheck`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- runDockerizationCheck: `trigger dockerizationcheck`
- runAuthorCheck: `trigger authorcheck`
- Checkstyle: `trigger checkstyle`
- PMD: `trigger pmd`
- TI-bootstrap: `trigger ti0`
- TI-bootstrap1: `trigger ti1`
- TI-bootstrap2: `trigger ti2`
- TI-bootstrap3: `trigger ti3`
- TI-bootstrap4: `trigger ti4`
- FunctionalTest1: `trigger ft1`
- FunctionalTest2: `trigger ft2`
- CodeBaseHash: `trigger codebasehash`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/36594)

### CODE REVIEW

**Review by vlica-harness — DISMISSED** [review]
> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/36594)* status: :shipit: complete! ✓

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 No issues Everything looks good
 [Details](https://app.embold.io/organization/gh/harness/repositories/7752dbb3099d2dba3b5d37f5931a5e4c/pull-request/36594)

**Review by nassergonzalez — APPROVED** [review]

File: 960-api-services/src/test/java/io/harness/azure/deployment/SlotContainerLogStreamerTest.java [hunk 1]
```diff
@@ -97,18 +97,18 @@ public void setup() {
...
-  public void testDDMMYYYYdateTimeparrternLogs() {
+  public void testDDMMYYYYdateTimeParrternLogs() {
     DeploymentSlot deploymentSlot = mock(DeploymentSlot.class);
     doReturn(Optional.of(deploymentSlot)).when(webClient).getDeploymentSlotByName(clientContext, SLOT_NAME);
     when(deploymentSlot.getContainerLogs())
         .thenReturn("14/08/2022 12:41:38.167 STDOUT - Site: test log 1".getBytes(StandardCharsets.UTF_8))
         .thenReturn("14/08/2022 12:45:38.167 STDOUT - Site: test log 2".getBytes(StandardCharsets.UTF_8));
     SlotContainerLogStreamer slotContainerLogStreamer =
-        new SlotContainerLogStreamer(clientContext, webClient, SLOT_NAME, logCallback);
+        new SlotContainerLogStreamer(clientContext, webClient, SLOT_NAME, logCallback);SlotContainerLogStreamerTest
```
Comment (vlica-harness) [inline 960-api-services/src/test/java/io/harness/azure/deployment/SlotContainerLogStreamerTest.java:107]: @tmacari-harness remove the additional "SlotContainerLogStreamerTest"...

**Review by tmacari-harness — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/36594)* status: 1 file(s) to review. 1. unresolved discussions. 0 of 1 approvals obtained. 

___
*[`960-api-services/src/test/java/io/harness/azure/deployment/SlotContainerLogStreamerTest.java` line 107 at r3](https://reviewable.io/reviews/harness/harness-core/36594#-NA49Fw3sAijZ84M0nWr-r3-107:-NA4F5ptEL7eIAid3LKt:bb6un1j) ([raw file](https://github.com/harness/harness-core/blob/2629098c9bf3552c33ad90f3edcfb1f11122088e/960-api-services/src/test/java/io/harness/azure/deployment/SlotContainerLogStreamerTest.java#L107)):*
<details><summary><i>Previously, vlica-harness (Vasile Lica) wrote…</i></summary><blockquote>

@tmacari-harness remove the additional "SlotContainerLogStreamerTest"... 
</blockquote></details>

Done

**Review by tmacari-harness — COMMENTED** [review]
> Reviewed 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/36594)* status: 1 file(s) to review. 1. unresolved discussions. 0 of 1 approvals obtained.

**Review by vlica-harness — COMMENTED** [review]
> Reviewed 1 of 1 files at r4, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/36594)* status: 1. unresolved discussions. 0 of 1 approvals obtained.

**Review by vlica-harness — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/36594)* status: :shipit: complete! ✓

**Pull request merged**